pages/platform/data-processing/01_CONCEPTS_Capabilities/guide.en-asia.md

```diff
@@ -63,6 +63,12 @@ It is not possible to interact with Apache Spark cluster via command line.
 
 - GRA (Gravelines in France) 
 
+### IP range to allow
+
+If you need to allow a list of IPs in your application network configuration, the ODP jobs will run on hosts
```

Comment:
Could you add this : "(for example to allow connections to one of your databases)"